lib/pages/card_page_jp.dart

```diff
@@ -81,18 +79,11 @@ class CardPageJP extends StatelessWidget {
               this.mail,
             ),
             SizedBox(
-              height: Constants.cardPaddingExtraSmall,
-            ),
-            _createSecondaryTexts(
-              Icons.location_on_outlined,
-              this.address,
-            ),
-            SizedBox(
-              height: Constants.cardPaddingExtraSmall,
+              height: Constants.cardPaddingSmall,
             ),
             _createSecondaryTexts(
               Icons.phone_android,
-              this.phone,
+              this.linkedin,
```

Comment:
LinkedIn is with two words, you should apply camel case on this throughout the code

Reply:
Fixed